.build-tools/pkg/metadataanalyzer/analyzer.template

```diff
@@ -129,12 +129,30 @@ func getYamlMetadata(basePath string, pkg string) *map[string]string {
 
 func checkMissingMetadata(yamlMetadata *map[string]string, componentMetadata mdutils.MetadataMap) []string {
 	missingMetadata := make([]string, 0)
+	if yamlMetadata == nil {
+		return missingMetadata
+	}
+
+  // use the built-in aws profile, don't need check in the metadata yaml file
+	usingAWSProfile := false
```

Comment:
i think we need update the metadataanalyzer: since now we use the built-in aws profile which will automatically append to it when generate the schema, we no longer check those related fields in the metadata.yaml file